sopt/src/main/scala/dagr/sopt/ArgTokenizer.scala

```diff
@@ -41,13 +44,15 @@ object ArgTokenizer {
 
 /** A class to tokenize a sequence of strings prior to option parsing.  A series of tokens will
   * be available.  If any errors are encountered, a failure will be present instead of the token.  */
-class ArgTokenizer(args: TraversableOnce[String]) extends Iterator[Try[ArgTokenizer.Token]] {
+class ArgTokenizer(val argFilePrefix: Option[String] = None,
+                   args: Seq[String]) extends Iterator[Try[ArgTokenizer.Token]] {
```

Comment:
Will fix.  This is a hangover from trying to make the Seq[String] and String* constructors match each other.